### PR TITLE
moved setup_marathon to content

### LINKED ISSUE #289: Add setup_marathon use case to content
now it is under `tests` - sets up a sample marathon env on a single Docker host ( according to http://viralkitty.com/marathon-and-docker/ )

it could be part of the actual content - WDYT?

### PR DESCRIPTION
**Pull request by SamMarkowitz**
fixes #289

### CODE REVIEW

**Comment by SamMarkowitz** [comment]
Not sure why the test is failing. I can't see how it's related to the my changes.

**Comment by Bonczidai** [comment]
build is not stable :(

```
22/10/15 07:27:36 [DEBUG] Event received: EVENT_ACTION_START Data is: {DESCRIPTION=Preparing to run action JAVA, CALL_ARGUMENTS={privateKeyFile=droplets_rsa, port=22, closeSession=false, username=core, arguments=null, host=188.166.39.246, agentForwarding=null, characterSet=UTF-8, pty=false, command=:, password=null, timeout=6000000}, PATH=0.3.0.0.0, STEP_NAME=null, STEP_TYPE=ACTION, TIMESTAMP=Thu Oct 22 07:27:35 UTC 2015, EXECUTION_ID=101600161, TYPE=EVENT_ACTION_START}
22/10/15 07:27:36 [DEBUG] Event received: EVENT_ACTION_END Data is: {DESCRIPTION=Action performed, PATH=0.3.0.0.0, RETURN_VALUES={returnCode=-1, returnResult=, STDERR=, exitStatus=-1, STDOUT=}, STEP_NAME=null, STEP_TYPE=ACTION, TIMESTAMP=Thu Oct 22 07:27:36 UTC 2015, EXECUTION_ID=101600161, TYPE=EVENT_ACTION_END}
```

feel free to restart it

**Comment by Bonczidai** [comment]
can you please add description for results, like:

```
# Results:
#   - CLEAR_CONTAINERS_ON_HOST_PROBLEM: clearing of host failed
#   - START_ZOOKEEPER_PROBLEM: failed to start Zookeeper container
....
```

**Comment by Bonczidai** [comment]
and please remove `outputs` description section in `run_container` since flow does not have any output

**Comment by SamMarkowitz** [comment]
I'm not sure I understand your last comment. Looks to me like the `run_container` flow has outputs:

```
  outputs:
    - container_ID
    - standard_err
```

**Comment by Bonczidai** [comment]
sorry, I meant `clear_containers`, it does not have `total_amount_of_images_deleted`

File: content/io/cloudslang/docker/containers/clear_containers.sl [hunk 1]
```diff
@@ -14,8 +14,12 @@
 #   - docker_password - Docker machine password
 #   - private_key_file - optional - path to private key file
 #   - timeout - optional - time in milliseconds to wait for the command to complete - Default: 6000000
+#   - port - optional - SSH port
 # Outputs:
 #   - total_amount_of_images_deleted - number of deleted images
+# Results:
+#   - SUCCESS - successful
+#   - FAILURE - otherwise
```
Comment (Bonczidai) [inline content/io/cloudslang/docker/containers/clear_containers.sl]: I don't think we need to mention default results here, they don't supply any additional info

**Comment by SamMarkowitz** [comment]
Okay. Cleaned up.

**Comment by Bonczidai** [comment]
:+1:

**Pull request merged**